webrender/src/device/gfx/device.rs

```diff
@@ -1309,6 +1328,7 @@ impl<B: hal::Backend> Device<B> {
                 &[self.bound_projection],
                 self.next_id
             );
+            self.rebind_descriptors = true;
```

Comment:
Do we even need `set_uniforms` function? We know when we start working on the new target, so we can just unconditionally update the projection uniforms and rebind the descriptor, instead of doing it lazily here